metaflow/plugins/cards/card_decorator.py

```diff
@@ -110,8 +206,9 @@ def task_finished(
     ):
         if not is_task_ok:
             return
+        component_strings = current.card._serialize_components(self._card_uuid)
         runspec = "/".join([current.run_id, current.step_name, current.task_id])
-        self._run_cards_subprocess(runspec)
+        self._run_cards_subprocess(runspec, component_strings)
```

Comment:
Is it on purpose that all cards are created serially? It seems there may be an opportunity to speed this up?